k8s/aat/common/vh/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```

Comment:
this should go into https://github.com/hmcts/shared-services-flux,

We don't have any docs in shared-services-flux currently but it's the exact same as far as I know

Reply:
just the identity yaml or this whole branch?

Reply:
everything in this PR